build-examples.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -e
+
+EXAMPLES=`find examples/ -maxdepth 1 -type d | tail -n +2`
+
+for example in $EXAMPLES; do
+	echo "Building snap for: $example"
+	(cd $example && \
```

Comment:
no need for `(` and `)`
Also instead of `cd $example` do `pushd $example` then `popd` at the end.